ci/tests/puppeteer/scenarios/login-success-jetty/script.json

```diff
@@ -0,0 +1,4 @@
+{
+  "dependencies": "core",
+  "projectName": "cas-server-webapp-jetty"
```

Comment:
I would suggest that we change this to "container" or "servletContainer" or "project" or "projectType", and then modify the shell script with a switch to just append tomcat/jetty/undertow to the end of the cas-server-webapp-{type} and handle others too. Seems more reasonable name to me and there would be no duplicating of the project prefixes (cas server webapp, etc), etc. It's almost like a DSL. 

You should also be able to get rid of the existing "type" field then as war or jar since those can be determined automatically. i.e. the matrix would be:

If container (or type) is (tomcat, jetty, undertow) then the type is a war.
If container is (starter,bootifulJar,jar, etc) or any other sensible tag you see fit, then auto-choose the starter project and set the type to be jar.

I'd like to keep all the conditions and checking in the shell script, and have configuration be as lean as possible. This should allow us in the future to build checks for boot-admin, config server, and other types of webapp (though these are just examples; no need browser tests there)